compiler/one-cmds/one-build.template.cfg

```diff
@@ -1,3 +1,6 @@
+[DEFAULT]
```

Comment:
This `DEFAULT` section seems somewhat different category compare to `one-build` and others.
Could you elaborate about this?

Reply:
`DEFAULT` section is a built-in section of `configparser`. It provides default values for all other sections. Below two config file are exactly the same meaning.
```cfg
[DEFAULT]
profiling_info=True

[one-build]
one-import-tf=True
one-import-tflite=False
one-import-bcq=False
one-optimize=True
one-quantize=False
one-pack=False
one-codegen=False

[one-import-tf]
input_path=inception_v3.pb
output_path=inception_v3.circle
input_arrays=input
input_shapes=1,299,299,3
output_arrays=InceptionV3/Predictions/Reshape_1
converter_version=v2

[one-optimize]
input_path=inception_v3.circle
output_path=inception_v3.opt.circle
```

```cfg
[one-build]
one-import-tf=True
one-import-tflite=False
one-import-bcq=False
one-optimize=True
one-quantize=False
one-pack=False
one-codegen=False

[one-import-tf]
input_path=inception_v3.pb
output_path=inception_v3.circle
input_arrays=input
input_shapes=1,299,299,3
output_arrays=InceptionV3/Predictions/Reshape_1
converter_version=v2
profiling_info=True // note here

[one-optimize]
input_path=inception_v3.circle
output_path=inception_v3.opt.circle
profiling_info=True // note here
```

Reply:
Thanks for detail :)
I'm not sure `DEFAULT` is the name for the purpose... need some time to think about this...

These are the questions arose...
- does the key-values exist in `DEFAULT` provide to all other sections? what happens if not?
- can other key-values move here?

Reply:
> does the key-values exist in DEFAULT provide to all other sections? what happens if not?

Sorry but I don't understand what the question wants to. `DEFAULT` section DOES provide its options with all other sections like above example.

> can other key-values move here?

As of now, I think logging and warning related options could be here.

Reply:
> Sorry but I don't understand what the question wants to. DEFAULT section DOES provide its options with all other sections like above example.

I think `all other sections` is dangerous...
While adding an option to each programs, `ALL` will be applied to all sections: which are commands.
We can only test by enabling all commands to support items in `DEFAULT`, if not command without that option may drop unsupported option, which means we cannot test individual commands for the support of that option unless others support this. 
Or  my understanding is incorrect?

Reply:
Another question: can each sections override `DEFAULT` ?
```
[DEFAULT]
profiling_info=True

[one-optimize]
profiling_info=False
```
I'm not sure this has some meaning but above will let all others has `profiling_info=True` except `one-optimize`

Reply:
This is something like what I understand at first seeing... although you may thing this is strange,
```
[DEFAULT]
profiling_info_value=True

[one-optimize]
profiling_info=profiling_info_value
```

then when we want to change it
```
[DEFAULT]
profiling_info_value=False

[one-optimize]
profiling_info=profiling_info_value
```

Reply:
@seanshpark 
I thought it would be possible for drivers to ignore unsupported options like `--profiling_info`. But, it won't be good a you said.

We can override the values in section at runtime. So, `profiling_info` key in the driver that doesn't need to it can be remove or set `False`.

> Another question: can each sections override DEFAULT ?

Yes. And as I said above, `profiling_info=False` can be moved to internal code rather than configuration file.

And, `DEFAULT` section name could be customized. If `DEFAULT` is a little bit weird, we can change it.